exercises/collatz-conjecture/example.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function CollatzConjecture() {}
+
+CollatzConjecture.prototype.steps = function (number) {
+  var count = 0;
+  if (number < 1) {
+    throw new Error('Only positive numbers are allowed');
+  }
+  getStepsCount(number);
+  function getStepsCount(n) {
+    if (n === 1) return 0;
+    var nextNumber = ( n % 2 === 0 ? n / 2 : (3 * n) + 1);
```

Comment:
Again, you don't need the exterior parens on this expression, or the parens around `(3 * n)`.